Bottom: b2fc6f16e8651478a4a3e16f19ac77eaf28d22b0
Top:    b8f0efe4fae6507de6744fc69fc65725ce922432
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-20 22:44:04 +0100

Refresh of gl-use-a-vbo-to-avoid-data-cop

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 35fa289..bdf9429 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -94,7 +94,7 @@ hidgl_flush_triangles (triangle_buffer *buffer)
   glUnmapBuffer (GL_ARRAY_BUFFER);
 
   glEnableClientState (GL_VERTEX_ARRAY);
-  glVertexPointer (2, GL_FLOAT, 0, NULL); // buffer->triangle_array);
+  glVertexPointer (3, GL_FLOAT, 0, NULL); // buffer->triangle_array);
   glDrawArrays (GL_TRIANGLES, 0, buffer->triangle_count * 3);
   glDisableClientState (GL_VERTEX_ARRAY);
